### DISCUSSION TITLE
PowerShell Durable Function Activity Bindings with Table Bindings

### DISCUSSION THREAD

**Discussion by Bionic711** [Azure/azure-functions-durable-extension#2904] [Q&A]
I am currently working on a Durable Function written in PowerShell and having issues passing the input of the activityTrigger to the rowKey of a Table input in the same activity. ```{input}``` from the name of the activity trigger allows it to run, and logs show it was passed ```["stringvalue"]```, but using the type as described in MSFT docs with the queue trigger does not. ```{activityTrigger}``` results in a 'not recognized as a named parameter' error. The interesting part is that when it reaches out to the table to pull the value when using ```{input}```, it says that the rowKey or partitionKey cannot be ''.

https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-table-input?tabs=isolated-process%2Ctable-api%2Cnodejs-v4&pivots=programming-language-powershell

Thus far I have only tested locally with Azurite as the emulator.

Any ideas how to get this to work?

**Comment by cgillum** [comment]
I think this question would be better tracked as a GitHub issue, so I've created one here: https://github.com/Azure/azure-functions-durable-extension/discussions/2904.
